ethicalads-theme/templates/ea/includes/publisher-testimonials.html

```diff
@@ -16,19 +16,19 @@ <h2 class="font-weight-bold mb-8">You're in great company</h2>
         <div class="card-body text-center">
 
           <!-- Icon -->
-          <div class="img-fluid mb-5 w-50 mx-auto">
-            <a href="https://routinehub.co/">
-              <img src="/theme/img/routinehub.svg" alt="RoutineHub" style="height: 100px" />
+          <div class="img-fluid mb-8 w-100 mx-auto">
+            <a href="https://eslint.org/">
+              <img src="/theme/img/eslint.svg" alt="ESLint" />
             </a>
           </div>
 
           {% if not short %}
             <!-- Text -->
-            <p class="text-gray-700 mb-5 small">RoutineHub is a community for sharing Siri shortcuts on iOS. EthicalAds helped them expand to meet demand during the last big iOS upgrade.</p>
+            <p class="text-gray-700 mb-5 small">ESLint, a static analyzer for JavaScript code, uses EthicalAds because of our commitment to privacy and transparency.</p>
```

Comment:
Burying the lead again :) 

```suggestion
            <p class="text-gray-700 mb-5 small">ESLint, a static analyzer for JavaScript code, uses EthicalAds because of our commitment to privacy and larger payouts.</p>
```